include/dxc/Support/HLSLOptions.td

```diff
@@ -271,6 +271,10 @@ def validator_version : Separate<["-", "/"], "validator-version">, Group<hlslcom
   HelpText<"Override validator version for module.  Format: <major.minor> ; Default: DXIL.dll version or current internal version.">;
 def print_after_all : Flag<["-", "/"], "print-after-all">, Group<hlslcomp_Group>, Flags<[CoreOption, HelpHidden]>,
   HelpText<"Print LLVM IR after each pass.">;
+def force_zero_store_lifetimes : Flag<["-", "/"], "force-zero-store-lifetimes">, Group<hlslcomp_Group>, Flags<[CoreOption]>,
```

Comment:
I think this option should be HelpHidden, but don't hold the PR for this.  I don't think we intend to keep this option long-term, it's a testing option while we decide whether undef stores are safe for downlevel.